Makefile

```diff
@@ -33,6 +57,8 @@ setup:
  		exit 1; \
 	fi
 
+	ln -sf ../../scripts/pre-commit .git/hooks/pre-commit
```

Comment:
I might want to overwrite my `pre-commit`... Maybe we shouldn't force re-overwrite it in that case.